src/polyfills/array.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/**
+ * Returns true if the element is in the array and false otherwise.
+ *
+ * @param {*} searchElement
+ * @param {number} fromIndex
+ * @returns {boolean}
+ */
+export function includes(value, fromIndex) {
```

Comment:
Does this need to be exported? If not, you can use a mock Window object with `install()` below to access it for testing.